uint/README.md

```diff
@@ -0,0 +1,8 @@
+# Big unsigned integer types
+
+Implementation of a various large-but-fixed sized unsigned integer types.
+The functions here are designed to be fast. There are optional `x86_64`
+implementations for even more speed, hidden behind the `x64_arithmetic`
+feature flag.
+
+Run tests with `cargo test --features=std,impl_quickcheck_arbitrary`.
```

Comment:
How can I improve this? :/

Reply:
Does https://doc.rust-lang.org/cargo/reference/manifest.html#the-required-features-field-optional help?

Reply:
I tried that and no, it's not really doing what I'd like. With:

```toml
[[test]]
name = "uint_tests"
required-features = ["std,impl_quickcheck_arbitrary"]
```
…I can run `cargo test` and all tests are skipped which is better than spewing errors, but now `cargo test --features=std,impl_quickcheck_arbitrary` also skips all tests.

What I'm looking for is a way to tell cargo "when running `cargo test` use this set of features: … … ".

Reply:
**UPDATE**: seems to be a [known limitation](https://github.com/rust-lang/cargo/issues/2911)